src/data/showcases.js

```diff
@@ -1202,6 +1202,15 @@ export const Showcases = [
     source: null,
     tags: ["daotool", "governance"],
   },
+  {
+    title: "1694.io",
+    description:
+        "1694.io is an open source zero click place to read CIP 1694. 1694.io is also place to see DRep timeline (pool.pm for drep), and for DRep to claim their profiles and engage with their delegators.",
```

Comment:
```suggestion
        "1694.io is an open source zero click place to read CIP 1694. 1694.io is also place to see DRep timeline (pool.pm for DReps), and for DRep to claim their profiles and engage with their delegators.",
```
Consistent capitalisation